bindata/assets/kube-apiserver/pod.yaml

```diff
@@ -44,30 +72,16 @@ spec:
     args:
         - |
           LOCK=/var/log/kube-apiserver/.lock
-          echo -n "Acquiring exclusive lock ${LOCK}"
-          exec {LOCK_FD}>${LOCK} && flock -n "${LOCK_FD}" || {
-            echo "$(date -Iseconds -u) kubelet did not terminate old kube-apiserver before new one" >> /var/log/kube-apiserver/lock.log
-            echo -n ": WARNING: kubelet did not terminate old kube-apiserver before new one."
-            # we didn't get an exclusive lock. We keep going with the risk to corrupt audit logs.
+          # At this point, we should be able to acquire the lock immediately since the setup container released it.
```

Comment:
That's not true. We might have timed out to get the lock.

Reply:
"we should be able" :-) wrong phrasing maybe

Reply:
# We should be able to acquire the lock immediatelly. If not, it means the init container has not released it yet and kubelet or CRI-O started container prematurely.